misc/format_server.py

```diff
@@ -30,8 +30,10 @@ def exec(self, cmd):
     def render_index(self):
         pulls = requests.get(
             f'https://api.github.com/repos/taichi-dev/taichi/pulls?state=open').json()
-        self.writeln(f'Click to auto-format PR')
+        self.writeln(f'Click to auto-format PR. <b>[Please do not click if the PR is not owned/reviewed by you.]</b>')
```

Comment:
They can if they rude, we may want to fliter PRs for current login user, may detect cookie.

Reply:
Also an administrator should have access to all PRs, how about to intergrate this into github actions?

Reply:
Please see #621 :-)